ElementX/Sources/Screens/RoomMemberDetails/View/RoomMemberDetailsScreen.swift

```diff
@@ -0,0 +1,138 @@
+//
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+struct RoomMemberDetailsScreen: View {
+    @ObservedObject var context: RoomMemberDetailsViewModel.Context
+    
+    var body: some View {
+        Form {
+            headerSection
+
+            // TODO: Uncomment when the feature is ready
+//            if !context.viewState.isAccountOwner {
+//                blockUserSection
+//            }
+        }
+        .scrollContentBackground(.hidden)
+        .background(Color.element.formBackground.ignoresSafeArea())
+        .alert(item: $context.blockUserAlertItem, actions: blockUserAlertActions(_:), message: blockUserAlertMessage(_:))
```

Comment:
I like the weird smiley faces, but they're not really necessary.
```suggestion
        .alert(item: $context.blockUserAlertItem, actions: blockUserAlertActions, message: blockUserAlertMessage)
```

Reply:
done